source/fundamentals/connection.txt

```diff
@@ -61,13 +62,15 @@ A MongoDB replica set deployment is a group of connected instances that
 store the same set of data. This configuration of instances provides data
 redundancy and high data availability.
 
-To connect to a replica set deployment, specify the host and port numbers of
-each instance, separated by a comma, and the replica set name as the value of
-the ``replicaSet`` parameter.
+To connect to a replica set deployment, specify the hostname and port numbers
+of each instance, separated by a comma, and the replica set name as the value
+of the ``replicaSet`` parameter in the connection string. You must use the
+protocol ``mongodb`` instead of ``mongodb+srv`` when you specify multiple
```

Comment:
A user does not decide to use `mongodb+srv` or `mongodb`, they are generally just "given" one. If they are using Atlas, then Atlas will default to `mongodb+srv` which uses an [SRV record](https://tools.ietf.org/html/rfc2782) to convey multiple addresses for the seed list.

Reply:
👍 I'll omit these two sentences.